src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -22,351 +22,358 @@
 
 tf.disable_v2_behavior()
 
-def get_augmented_data(images, annotations, output_height, output_width, resize_only):
+_DEFAULT_AUG_PARAMS = {
+  'max_hue_adjust' : 0.05,
+  'max_brightness' : 0.05,
+  'max_contrast' : 1.25,
+  'max_saturation' : 1.25,
+  'flip_pr' : True,
```

Comment:
It doesn't look like this is getting used. Also I don't understand what "pr" means.

Reply:
I need to remove it